src/shared/redux/slices/settingsSlice.ts

```diff
@@ -1,15 +1,24 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { VersionDetails } from '../../../main/versionDetails';
 import { SupportedLangCode } from '../../../renderer/internationalisation/i18next';
 import { RootState } from '../rootReducer';
 
 type Settings = {
   defaultDirPath: string;
   currentLocale: SupportedLangCode;
+  versionDetails: {
+    version: string;
+    isUpToDate: boolean;
+  };
```

Comment:
Since you have the `VersionDetails` interface imported anyway and use it as payload type in the `setVersionDetails` you can also use it as a type here.

Reply:
Fixed, unfortunately the review went stale